cloudbuild.yaml

```diff
@@ -32,5 +32,14 @@ steps:
       - --cache=true
       - --cache-ttl=48h
     waitFor: ['-']
```

Comment:
Do we want to also publish to dockerhub and add to changesets? It feels like changesets is really convenient for js

Reply:
I plan on deploying this to internal infrastructure only. The infra would pull a specific commit or `develop` so we don't strictly need changesets for this. We can always add it later.